tracer/test/snapshots/Security.AspNetCore5AsmData.__test=blocking-user_enableSecurity=True_expectedStatusCode=403_url=_user.verified.txt

```diff
@@ -0,0 +1,86 @@
+﻿[
```

Comment:
I think this snapshot is no longer used after refactoring to the class-based configurations

Reply:
removed in b9a46c5f6